### PR TITLE
Compile Botan with --disable-asm in MSAN builds

### PR DESCRIPTION
**Pull request by guidovranken**

### CODE REVIEW

**Comment by guidovranken** [comment]
A recent change in Botan necessitates compiling without assembly language optimizations to prevent memorysanitizer false positives: https://github.com/randombit/botan/issues/4265#issuecomment-2257074148

File: projects/libecc/build.sh [hunk 1]
```diff
@@ -39,7 +39,12 @@ export CXXFLAGS="$CXXFLAGS -DCRYPTOFUZZ_LIBECC"
 cd $SRC/botan
 if [[ $CFLAGS != *-m32* ]]
 then
-    ./configure.py --cc-bin=$CXX --cc-abi-flags="$CXXFLAGS" --disable-shared --disable-modules=locking_allocator,x509 --build-targets=static --without-documentation
+    if [[ $CFLAGS != *sanitize=memory* ]]
+    then
+        ./configure.py --cc-bin=$CXX --cc-abi-flags="$CXXFLAGS" --disable-shared --disable-modules=locking_allocator --build-targets=static --without-documentation
```
[the remark this answers is not in the document]
Comment (guidovranken) [reply projects/libecc/build.sh]: Thanks I've disabled it.

**Comment by guidovranken** [comment]
I don't know what the problem is with the coverage build. The build log is truncated.

**Comment by guidovranken** [comment]
> I don't know what the problem is with the coverage build. The build log is truncated.

Can someone help, tell me what the error is? @DavidKorczynski @jonathanmetzman

**Comment by jonathanmetzman** [comment]
> > I don't know what the problem is with the coverage build. The build log is truncated.
> 
> Can someone help, tell me what the error is? @DavidKorczynski @jonathanmetzman

It's cut off for me too :-)

**Comment by guidovranken** [comment]
@jonathanmetzman @DavidKorczynski can you merge it? I'll fix any coverage build failures later.

**Pull request merged**